### PR TITLE
autobump: track more formulae

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by samford — CHANGES_REQUESTED** [review]
I've gone through the added formulae in this PR and identified the ones that can't be added at this point (review comments below). For future reference, these are: ~`hpack`~, ~`rover`~, `scalapack`, ~and `triangle`~.

I've also identified formulae that need (or would benefit from) `livecheck` block changes and opened a few related PRs. The `livecheck` PRs should be merged before this PR. Edit: These PRs have been merged.

**Review by MikeMcQuaid — COMMENTED** [review]
Thanks for adding more of these @chenrui333 and checking them over @samford!

File: .github/workflows/autobump.yml [hunk 1]
```diff
@@ -695,14 +763,19 @@ env:
     traefik@1
     trafficserver
     tree-sitter
+    triangle
```
[the remark this answers is not in the document]
Comment (samford) [reply .github/workflows/autobump.yml:766]: Repology split this into a separate `triangle-art` entry, so this is fine now:

```
==> triangle
Current formula version:  1.2.5
Latest livecheck version: 1.2.5
Latest Repology version:  present only in Homebrew
```

File: .github/workflows/autobump.yml [hunk 2]
```diff
@@ -333,6 +346,8 @@ env:
     himalaya
     hive
     hivemind
+    hledger
+    hpack
```
[the remark this answers is not in the document]
Comment (samford) [reply .github/workflows/autobump.yml:350]: This has been resolved on Repology, so it's fine now.

```
==> hpack is up to date!
Current formula version:  0.34.7
Latest livecheck version: 0.34.7
Latest Repology version:  0.34.7
```

File: .github/workflows/autobump.yml [hunk 3]
```diff
@@ -587,6 +640,7 @@ env:
     rke
     rollup
     rosa-cli
+    rover
```
[the remark this answers is not in the document]
Comment (samford) [reply .github/workflows/autobump.yml:643]: This has been resolved on Repology, so it's fine now (the newer livecheck version is correct).

```
==> rover
Current formula version:  0.5.0
Latest livecheck version: 0.5.1
Latest Repology version:  0.5.0
```

**Review by samford — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Pull request merged**